conftest.py

```diff
@@ -350,6 +300,85 @@ def get_driver(self):
         options.add_argument("--js-flags=--expose-gc")
         return Chrome(options=options)
 
+    def collect_garbage(self):
+        self.driver.execute_cdp_cmd("HeapProfiler.collectGarbage", {})
+
+
+class NodeWrapper(SeleniumWrapper):
+    browser = "node"
+
+    def init_node(self):
+        os.chdir("build")
+        self.p = pexpect.spawn(
+            f"node --expose-gc ../node_test_driver.js {self.base_url}", timeout=60
+        )
+        self.p.setecho(False)
+        os.chdir("..")
+
+    def get_driver(self):
+        self._logs = []
+        self.init_node()
+
+        class NodeDriver:
+            def __getattr__(self, x):
+                raise Exception("BLAHH", x)
+                # pytest.skip("unsupported configuration")
```

Comment:
Please remove commend and make the exception message a bit more explicit. Maybe a `NotImplementedError` instead ?